Makefile

```diff
@@ -761,7 +761,7 @@ build/check-doc.build-stamp: doc/build/html/index.html doc/build/html/htmlcov/in
 			-e "No module named '(SpaceAdapter|Optimizer)Factory'" \
 			-e '^make.*resetting jobserver mode' \
 		| grep -v '^\s*$$' \
-		| if grep .; then echo "Errors found in doc build. Check doc/build/log.txt for details."; exit 1; else exit 0; fi
+		| if grep .; then echo "Errors found in doc build. Check doc/build/log.txt for details."; cat doc/build/log.txt; exit 1; else exit 0; fi
```

Comment:
Might want to remove this change, but it's also kinda handy for debugging future failures since I can't reproduce locally.

Reply:
Gonna leave this for now.